### PR TITLE
Website changes for the patches `v17.0.3`, `v16.0.5`, `v15.0.5`

### PR DESCRIPTION
**Pull request by GuptaManan100**
### Description

This PR does the website changes for the patches `v17.0.3`, `v16.0.5`, `v15.0.5`.

### CODE REVIEW

**Review by frouioui — APPROVED** [review]

File: content/en/docs/15.0/get-started/local.md [hunk 1]
```diff
@@ -88,7 +88,8 @@ Download the [latest binary release](https://github.com/vitessio/vitess/releases
 * Please use release [15.0.3](https://github.com/vitessio/vitess/releases/tag/v15.0.3) or later.
 
 ```sh
-version=15.0.4
+version=15.0.5
```
Comment (ajm188) [inline content/en/docs/15.0/get-started/local.md:91]: can you update the version pairs in the makefile as well?

Comment (mattlord) [reply content/en/docs/15.0/get-started/local.md:91]: I believe we planned to do that in the 18.0-RC docs PR.

File: content/en/docs/17.0/get-started/local.md [hunk 2]
```diff
@@ -65,8 +65,8 @@ Download the [latest binary release](https://github.com/vitessio/vitess/releases
 * Ubuntu is the only fully supported OS, for another OS please [build Vitess by yourself](/docs/contributing) or use the Docker images.
 
 ```sh
-version=17.0.2
-file=vitess-${version}-96ac0a6.tar.gz
+version=17.0.3
+file=vitess-${version}-dced85dd0f.tar.gz
```
Comment (frouioui) [inline content/en/docs/17.0/get-started/local.md]:
```suggestion
file=vitess-${version}-dced85d.tar.gz
```

File: content/en/docs/16.0/get-started/local.md [hunk 3]
```diff
@@ -89,8 +89,8 @@ Download the [latest binary release](https://github.com/vitessio/vitess/releases
 * Ubuntu is the only fully supported OS, for another OS please [build Vitess by yourself](/docs/contributing) or use the Docker images.
 
 ```sh
-version=16.0.4
-file=vitess-${version}-87ea735.tar.gz
+version=16.0.5
+file=vitess-${version}-fd462ffb10.tar.gz
```
Comment (frouioui) [inline content/en/docs/16.0/get-started/local.md]:
```suggestion
file=vitess-${version}-fd462ff.tar.gz
```

File: content/en/docs/15.0/get-started/local.md [hunk 4]
```diff
@@ -88,8 +88,8 @@ Download the [latest binary release](https://github.com/vitessio/vitess/releases
 * Please use release [15.0.3](https://github.com/vitessio/vitess/releases/tag/v15.0.3) or later.
 
 ```sh
-version=15.0.4
-file=vitess-${version}-72d943e.tar.gz
+version=15.0.5
+file=vitess-${version}-1871cee516.tar.gz
```
Comment (frouioui) [inline content/en/docs/15.0/get-started/local.md]:
```suggestion
file=vitess-${version}-1871cee.tar.gz
```

**Review by frouioui — APPROVED** [review]

**Review by harshit-gangal — APPROVED** [review]

File: Makefile [hunk 5]
```diff
@@ -42,7 +42,7 @@ check-all-links: clean build link-checker-setup
 	bin/htmltest --conf .htmltest.external.yml
 
 ifndef COBRADOC_VERSION_PAIRS
-export COBRADOC_VERSION_PAIRS="main:18.0,v17.0.2:17.0,v16.0.4:16.0,v15.0.4:15.0"
+export COBRADOC_VERSION_PAIRS="main:19.0,v18.0.0:18.0,v17.0.3:17.0,v16.0.5:16.0,v15.0.5:15.0"
```
Comment (ajm188) [inline Makefile:45]: sorry i just noticed this, the 18.0 tag is `v18.0.0-rc1` so this is currently broken

**Pull request merged**